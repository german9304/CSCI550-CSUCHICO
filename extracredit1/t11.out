Sum Red Nova 412560
Sum Star Trace 235470
red_nova_stations: 82
max_weight red nova 1018
----------red nova----------
parent_array
0 81 63 16 14 39 76 40 75 18 79 14 28 23 0 72 28 26 68 80 12 80 41 12 76 10 72 60 44 54 3 26 45 18 39 17 67 67 36 79 0 39 70 35 57 46 59 26 21 29 66 7 70 43 15 24 11 51 12 33 6 58 53 7 80 75 28 4 13 43 54 36 46 16 71 27 17 20 37 13 8 72 
distance_array
0 9379 1233 3063 299 6262 11839 496 13406 6145 6089 954 3307 4910 228 8831 2788 10250 5504 13865 3618 14285 7143 4188 11466 6586 9283 12950 2336 9636 4081 9332 8479 6078 6090 10783 1340 803 1940 5594 228 6199 9745 11075 1366 7889 7527 9828 14816 9890 2928 1162 10582 11357 9375 12245 1853 1331 4209 6635 12279 4743 12071 621 14669 13173 2713 465 4973 11267 9573 1509 8425 2892 2398 12972 10947 4338 1462 5309 13765 9279 
max_weight star trace 1019
---------star trace---------
parent_array
0 14 20 31 23 35 33 31 25 22 20 33 13 31 26 1 23 14 0 38 18 3 20 22 15 40 23 38 23 22 6 18 28 22 36 24 20 18 13 23 28 
distance_array
0 4878 2214 1715 2790 7950 2528 1934 5257 2367 1736 3029 1933 1816 4058 5831 2983 4944 659 2540 1195 2228 1835 2491 6416 4415 3138 2738 3279 2823 3080 1517 3557 2094 2120 7062 1866 958 2016 3035 3781 
total: 648030
total_values: 
663646
659161
