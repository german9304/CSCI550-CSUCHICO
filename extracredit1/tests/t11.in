82
0 14 228
0 40 228
1 81 100
2 63 612
3 16 275
3 30 1018
4 14 71
4 67 166
5 39 668
6 60 440
6 76 892
7 40 268
7 51 666
7 63 125
8 75 434
8 80 359
9 18 641
10 25 497
10 79 780
11 14 726
11 56 899
12 20 311
12 23 881
12 28 971
12 58 902
13 23 722
13 68 63
13 79 399
15 54 544
15 72 406
16 28 452
16 73 104
17 26 967
17 35 533
17 76 697
18 33 574
18 68 531
19 80 100
20 77 720
21 48 531
21 80 520
22 41 944
24 55 779
24 76 519
26 31 49
26 47 545
26 72 858
27 60 671
27 75 22
28 44 970
28 66 377
29 49 254
29 54 261
32 45 590
33 59 557
34 39 496
35 43 292
36 38 600
36 67 875
36 71 169
37 67 338
37 78 659
39 41 605
39 79 285
42 70 172
43 53 282
43 69 192
44 57 35
45 46 362
46 59 892
46 72 898
50 66 215
51 57 169
52 70 1009
53 62 714
54 70 198
58 61 534
64 80 904
65 75 201
71 74 889
72 81 854
41
0 18 659
1 14 820
1 15 953
2 20 1019
3 21 513
3 31 198
4 23 299
5 35 888
6 30 552
6 33 434
7 31 417
8 25 842
9 22 532
10 20 541
11 33 935
12 13 117
13 31 299
13 38 200
14 17 886
14 26 920
15 24 585
16 23 492
18 20 536
18 31 858
18 37 299
19 38 524
20 22 640
20 36 671
22 23 656
22 29 988
22 33 259
23 26 647
23 28 788
23 39 544
24 35 646
25 40 634
27 38 722
28 32 278
28 40 502
34 36 254
9 25 1543 186
59 17 1083 301
