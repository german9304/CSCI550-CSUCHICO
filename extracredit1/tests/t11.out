59 17 659161
